src/examples/rest/client.cpp

```diff
@@ -78,7 +85,7 @@ int main(int argc, char **argv) {
 
   VERBOSE << "Connecting to server: " + serverHost + " port: " << port << std::endl;
   httplib::Client client(serverHost.c_str(), port);
-  client.set_timeout_sec(30);  // The time it waits for a network connection.
+  //client.set_timeout_sec(30);  // The time it waits for a network connection.
```

Comment:
@Zbysekz did you find out if there's a replacement for the `httplib.set_timeout_sec()`, or if we just stop using that function?

Reply:
Asked question [https://github.com/yhirose/cpp-httplib/issues/642](https://github.com/yhirose/cpp-httplib/issues/642)

Reply:
We have an answer: `set_connection_timeout()` is the substitute.
@breznak this PR is closed maybe starting another? I am not sure about consequences, or what timeout is by default

Reply:
nice! sorry for merging early. Yes, we can start a new PR where you can revert commit `606cb75d00cb376` and then change it to the new signature. 

> I am not sure about consequences, or what timeout is by default 

nor do i. our tests passed even without it. maybe in the server it makes sense, in the unit tests 30sec is imho too much.